### PR TITLE
New package: WHTA.Tamara version 2023.0.8.3

### PR DESCRIPTION
**Pull request by lars-nvhgroup**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/130026)

### CODE REVIEW

**Comment by lars-whta** [comment]
> Hello @lars-whta,
> 
> During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?
> 
> This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.
> 
> Please investigate a fix and resubmit the Pull Request.
> 
> You may also try using the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) to determine the proper installer type. If the installer type is MSIX, MSI, or a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass. If it's a .exe installer of an unknown type, you will need to search to determine the proper switches for Silent and SilentWithProgress.
> 
> Template: msftbot/validationError/installation/unattended

I am quite sure it works silently. Why does it time-out (or similar)?
Does the test environment have enough disk-, CPU- and RAM resources?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/w/WHTA/Tamara/2023.0.8.3/WHTA.Tamara.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WHTA.Tamara
+PackageVersion: 2023.0.8.3
+InstallerType: exe
+InstallerSwitches:
+  Silent: -mode silent -agreeToLicense yes
+  SilentWithProgress: -mode silent -agreeToLicense yes
```
Comment (stephengillie) [inline manifests/w/WHTA/Tamara/2023.0.8.3/WHTA.Tamara.installer.yaml:9]:
```suggestion
  SilentWithProgress: -mode silent -agreeToLicense yes
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (lars-whta) [reply manifests/w/WHTA/Tamara/2023.0.8.3/WHTA.Tamara.installer.yaml:9]: Thanks

**Pull request closed without merging**